tests/conftest.py

```diff
@@ -444,33 +445,44 @@ def __exit__(self, type, value, traceback):
             dag.__exit__(type, value, traceback)
             if type is None:
                 dag.clear()
-                self.dag_run = dag.create_dagrun(
-                    run_id=self.kwargs.get("run_id", "test"),
-                    state=self.kwargs.get('state', State.RUNNING),
-                    execution_date=self.kwargs.get('execution_date', self.kwargs['start_date']),
-                    start_date=self.kwargs['start_date'],
-                )
+
+        def _set_default_args(self, kwargs, defaults):
+            for k, v in defaults.items():
+                if k not in kwargs:
+                    kwargs.setdefault(k, v)
+
+        @provide_session
+        def make_dagmodel(self, session=None, **kwargs):
+            dag = self.dag
+            defaults = dict(dag_id=dag.dag_id, next_dagrun=dag.start_date, is_active=True)
+            self._set_default_args(kwargs, defaults)
```

Comment:
I think this can be instead

```suggestion
            kwargs = {**defaults, **kwargs}
```

[PEP 448](https://www.python.org/dev/peps/pep-0448/#specification):

> The keys in a dictionary remain in a right-to-left priority order, so `{**{'a': 1}, 'a': 2, **{'a': 3}}` evaluates to `{'a': 3}`. There is no restriction on the number or position of unpackings.

Reply:
This is so apt! Thanks!